src/lib/network/clients/save_network.rb

```diff
@@ -220,6 +204,39 @@ def copy_from_instsys
       nil
     end
 
+    WICKED_DHCP_PATH = "/var/lib/wicked/".freeze
+    WICKED_DHCP_FILES = ["duid.xml", "iaid.xml", "lease*.xml"].freeze
+    DHCPv4_PATH = "/var/lib/dhcp/".freeze
+    DHCPv6_PATH = "/var/lib/dhcp6/".freeze
+    DHCP_FILES = "*.leases".freeze
+
+    def copy_dhcp_info
```

Comment:
Hmm, the whole function looks like copy&paste. 3x the same.

Reply:
Yep it smells so and probably could be handled with loop with dirs and files, but as I was not sure about removing the copy /var/lib/dhcp files I let it like duplicated and clearly separated